api/src/main/java/org/openmrs/util/DoubleRange.java

```diff
@@ -77,6 +77,39 @@ public void setLow(Double low) {
 		this.low = low == null ? new Double(Double.NEGATIVE_INFINITY) : low;
 	}
 	
+	/**
+	 * @return Returns the closedLow 
+	 * 
+	 */
+	public boolean getclosedLow () {
+		return closedLow;
+	}
+	
+	/**
+	 * @param closedLow to set
+	 * 
+	 */
+	public void closedLow(boolean closedLow) {
+		this.closedLow = closedLow == true;
+	}
+	
+	/**
+	 * @return Returns the closedHigh 
+	 * 
+	 */
+	public boolean getclosedHigh () {
+		return closedHigh;
+	}
+	
+	/**
+	 * @param closedHigh to set
+	 * 
+	 */
+	public void closedHigh(boolean closedHigh) {
+		this.closedHigh = closedHigh == false;
```

Comment:
This seems wrong. when the method parameter closedHigh is false you will be assigning true to the this.closedHigh. Is that correct _